datalab/mlalpha/_confusion_matrix.py

```diff
@@ -11,57 +11,99 @@
 # the License.
 
 
-from plotly.offline import iplot
+import google.cloud.ml as ml
+import numpy as np
+import json
+import matplotlib.pyplot as plt
+import pandas as pd
+from sklearn.metrics import confusion_matrix
+
+import datalab.bigquery as bq
+import datalab.data as data
 
 
 class ConfusionMatrix(object):
   """Represents a confusion matrix."""
 
-  def __init__(self, predicted_labels, true_labels, counts):
-    """Initializes an instance of a ComfusionMatrix. the length of predicted_values,
-       true_values, count must be the same.
+  def __init__(self, cm, labels):
+    """
+    Args:
+      cm: a 2-dimensional matrix with row index being target, column index being predicted,
+          and values being count.
+      labels: the labels whose order matches the row/column indexes.
+    """
+    self._cm = cm
+    self._labels = labels
 
+  @staticmethod
+  def from_csv(input_csv, headers=None, schema_file=None):
+    """Create a ConfusionMatrix from a csv file.
     Args:
-      predicted_labels: a list of predicted labels.
-      true_labels: a list of true labels.
-      counts: a list of count for each (predicted, true) combination.
+      input_csv: Path to a Csv file (with no header). Can be local or GCS path.
+      headers: Csv headers. If present, it must include 'target' and 'predicted'.
+      schema_file: Path to a JSON file containing BigQuery schema. Used if "headers" is None.
```

Comment:
Add "If present, it must include 'target' and 'predicted'." to the schema_file line too or just say there has to be two columns named target and predicted before the args list.